src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -106,8 +107,15 @@ public Command parseCommand(String userInput) throws ParseException {
             return new SwitchCommand();
 
         case ArchiveCommand.COMMAND_WORD:
-            return new ArchiveCommandParser().parse(arguments);
-
+            if (StatusBarFooter.isArchiveBook()) {
+                throw new ParseException("You are currently in the archived book!\nDid you mean 'unarchive'?");
+            }
+            return new ArchiveCommandParser("ARCHIVE").parse(arguments);
+        case ArchiveCommand.ALT_COMMAND_WORD:
+            if (!StatusBarFooter.isArchiveBook()) {
+                throw new ParseException("You are currently in the default book!\nDid you mean 'archive'?");
+            }
+            return new ArchiveCommandParser("UNARCHIVE").parse(arguments);
```

Comment:
Try not to use magic constant.

Reply:
Good point on the magic constants. I could have used `ArchiveCommand.COMMAND_WORD` and `ArchiveCommand.ALT_COMMAND_WORD` instead.

Thanks for fixing the test cases, no idea how that happened lmao.

Will update UG, then merge if @fantablack has no issues with it.